examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -122,6 +124,14 @@ def run_and_monitor_progress(
     outs = [[], []]
     ipu_monitoring: List[str] = []
 
+    def kill_process(proc_pid):
+        process = psutil.Process(proc_pid)
```

Comment:
Killing a process is not that simple :)
In my past job, I wrote a few-screen function to kill a process that was always effective :)